podcasts/podcasts-app/src/webapp/podcasts/podOptions.jsp

```diff
@@ -42,7 +42,7 @@
     </div>
     <br />
 
-  <table class="listHier lines nolines nocolor" cellpadding="0" cellspacing="0" border="0"
+  <table class="table" cellpadding="0" cellspacing="0" border="0"
```

Comment:
How about getting rid of these old and deprecated settings like cellpading, cellspacing, and border?

Do you want the table to have borders or stripes?